package.json

```
@@ -50,7 +48,9 @@
   },
   "peerDependencies": {
     "expo": ">=47.0.0",
-    "expo-build-properties": ">=0.5.1"
+    "expo-build-properties": ">=0.5.1",
+    "react": "19.1.0",
+    "react-native": "0.81.0"
```

I think it would be better to use "Compatible with version" versioning.
```suggestion
    "react": "^19.1.0",
    "react-native": "^0.81.0"
```

---

We need to issue static versions here since with a caret react will use 19.1.1 which is not compatible with react-native-renderer 19.1.0 which is what the latest react-native uses